content/en/examples/controllers/job.yaml

```diff
@@ -7,8 +7,8 @@ spec:
     spec:
       containers:
       - name: pi
-        image: perl:5.34
-        command: ["perl",  "-Mbignum=bpi", "-wle", "'print bpi(2000)'"]
+        image: perl:5.34.0
+        command: ["perl",  "-Mbignum=bpi", "-wle", "print bpi(2000)"]
```

Comment:
Hi @mhus . Do you need single quotes around the `print` command (I'm not sure why the quotes are needed)? If not, would you remove the single quotes?
The other changes look good to me. Thank you!